src/Tools/dotnet-trace/CommandLine/Commands/ListProfilesCommandHandler.cs

```diff
@@ -15,6 +15,18 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal sealed class ListProfilesCommandHandler
     {
+        /// <summary>
+        /// Indicates diagnostics messages from DiagnosticSourceEventSource should be included.
+        /// </summary>
+        /// <remarks>See: https://github.com/dotnet/corefx/blob/master/src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/DiagnosticSourceEventSource.cs</remarks>
+        private const long DiagnosticSourceKeywords_Messages = 0x1;
```

Comment:
can you make this an enum instead? Keywords are provider specific so I'd prefer to not leave them as top-level fields